### PR TITLE
Hellanmaa Winter Units

### PR DESCRIPTION
**Pull request by TWatson1162**

### CODE REVIEW

File: addons/polkbaskera/CfgVehicles_Backpacks.hpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+	class B_AssaultPack_rgr;
+	class CLASS(B_AssaultPack_rgr_AT): B_AssaultPack_rgr {
+		class TransportWeapons {};
+		class TransportItems {};
+		class TransportMagazines {RPG7_F,2};
```
Comment (BrettMayson) [inline addons/polkbaskera/CfgVehicles_Backpacks.hpp]:
```suggestion
		class TransportMagazines {
			class _xx_RPG7_F {
				magazine = "RPG7_F";
				count = 2;
			};
		};
```

File: addons/polkbaskera/CfgVehicles_Units.hpp [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	weapons[] = {"arifle_AKM_F", "launch_RPG7_F", "Throw", "Put"};
+	respawnWeapons[] = {"arifle_AKM_F", "launch_RPG7_F", "Throw", "Put"};
+	magazines[] = {R5("30Rnd_762x39_Mag_Green_F"), R3("RPG7_F"), R2("SmokeShell")};
+	respawnMagazines[] = {R5("30Rnd_762x39_Mag_Green_F"), R3("RPG7_F"), R2("SmokeShell")};
+	linkedItems[] = {"WSLV_Platecarrier_GRN_NF","G_Balaclava_oli", "ItemMap", "ItemCompass", "ItemWatch", "ItemRadio"};
+	respawnLinkedItems[] = {"WSLV_Platecarrier_GRN_NF","G_Balaclava_oli", "ItemMap", "ItemCompass", "ItemWatch", "ItemRadio"};
+	Items[] = {R4("ACE_elasticBandage"), R4("ACE_packingBandage"), R2("ACE_quikclot")};
+	RespawnItems[] = {R4("ACE_elasticBandage"), R4("ACE_packingBandage"), R2("ACE_quikclot")};
+	uniformClass = "CUP_I_B_PMC_Unit_29";
+	Backpack = "B_AssaultPack_rgr_AT";
```
Comment (BrettMayson) [inline addons/polkbaskera/CfgVehicles_Units.hpp:44]:
```suggestion
	Backpack = QCLASS(B_AssaultPack_rgr_AT);
```

**Review by BrettMayson — CHANGES_REQUESTED** [review]

File: addons/polkbaskera/CfgVehicles_Units.hpp (hunk 2 above)
Comment (BrettMayson) [inline addons/polkbaskera/CfgVehicles_Units.hpp]: Use the backpack you've defined with the AT rounds
```suggestion
	backpack = QCLASS(B_AssaultPack_rgr_AT);
```

File: addons/polkbaskera/CfgVehicles_Units.hpp [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+};
+
+class CLASS(B_ATRifleman_F): CLASS(B_Rifleman_F) {
+	displayname = "AT Rifleman";
+	icon = "iconManAT";
+	role = "MissileSpecialist";
+	weapons[] = {"arifle_AKM_F", "launch_RPG7_F", "Throw", "Put"};
+	respawnWeapons[] = {"arifle_AKM_F", "launch_RPG7_F", "Throw", "Put"};
+	magazines[] = {R5("30Rnd_762x39_Mag_Green_F"), R3("RPG7_F"), R2("SmokeShell")};
+	respawnMagazines[] = {R5("30Rnd_762x39_Mag_Green_F"), R3("RPG7_F"), R2("SmokeShell")};
```
Comment (BrettMayson) [inline addons/polkbaskera/CfgVehicles_Units.hpp]: Only need to add one, for the weapon
```suggestion
	magazines[] = {R5("30Rnd_762x39_Mag_Green_F"), "RPG7_F", R2("SmokeShell")};
	respawnMagazines[] = {R5("30Rnd_762x39_Mag_Green_F"), "RPG7_F", R2("SmokeShell")};
```

File: addons/polkbaskera/CfgVehicles_Backpacks.hpp [hunk 4]
```diff
@@ -0,0 +1,6 @@
+	class B_AssaultPack_rgr;
+	class CLASS(B_AssaultPack_rgr_AT): B_AssaultPack_rgr {
```
Comment (BrettMayson) [inline addons/polkbaskera/CfgVehicles_Backpacks.hpp]:
```suggestion
	class CLASS(B_AssaultPack_rgr_AT): B_AssaultPack_rgr {
		dlc = QUOTE(PREFIX); 
		scope = 1; 
		scopeCurator = 1;
```

**Review by BrettMayson — CHANGES_REQUESTED** [review]

**Comment by TWatson1162** [comment]
ngl not sure what I did, but I did correctly apply all the changes requested -- however I tried to copy it for other roles (ex: CLS, Demo, Engi)

**Comment by TWatson1162** [comment]
yah that didn't work

**Comment by TWatson1162** [comment]
k last push of the night, no more stupid piecemeal updates

**Pull request merged**